### ISSUE TITLE
RTL: relax FSharp.Core dependency from >= 4.7.1 to >= 4.7.0 in 

### ISSUE THREAD

**Issue by Zaid-Ajaj** [Shmew/Fable.Jester#4]
Would be possible to relax the FSharp.Core dependency constraint from `>= 4.7.1` to `>= 4.7.0` in `Fable.ReactTestingLibrary@0.7.0`? I am getting compatibility warnings when I install the library:
```
warn : Detected package downgrade: FSharp.Core from 4.7.1 to 4.7.0. Reference the package directly from the project to select a different version. 
warn :  Tests -> Fable.ReactTestingLibrary 0.7.0 -> FSharp.Core (>= 4.7.1 && < 5.0.0)
warn :  Tests -> FSharp.Core (>= 4.7.0)
```

**Comment by Shmew** [comment]
That's odd, I just have it set in [paket.dependencies](https://github.com/Shmew/Fable.Jester/blob/master/paket.dependencies#L52) which should resolve to `>= 4.7 && < 5`

**Comment by Shmew** [comment]
Fixed in 0.7.1

**Comment by Zaid-Ajaj** [comment]
Awesome! thanks a lot :heart:
